pkgs/development/web/insomnium/default.nix

```diff
@@ -0,0 +1,80 @@
+{ pkgs, stdenv, lib, makeDesktopItem, makeWrapper, fetchurl, appimageTools
+, undmg, ... }:
+let
+  pname = "insomnium";
+  version = "0.2.3-a";
+  appname = "Insomnium";
+  name = "${appname}.Core-${version}";
+  meta = with lib; {
+    description =
+      "Fast local API testing tool that is privacy-focused and 100% local";
+    homepage = "https://github.com/ArchGPT/insomnium";
+    license = licenses.mit;
+    maintainers = [ "nephalemsec" ];
```

Comment:
Didn't know such a thing existed, I don't really mind who's name is there as long as the package is available for people to use.

Reply:
It would be great if you could also keep this package up to date, when a new version is available upstream, though.

Reply:
Yes, but you should add yourself in it instead of a string with your username. It enables some features within ofborg and also give meta data about you on the package search.

Refer to https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers